src/imx_rt/rt1060/README.md

```diff
@@ -1 +1,238 @@
-# TODO
+# OpenThread on NXP RT1060 (host) + K32W061 (rcp) Example
```

Comment:
Please replace with:
OpenThread on NXP RT1060 (host) + K32W061 (rcp) Example **(Experimental support)**

Reply:
I did the modification in the latest PR commit.